### ISSUE TITLE
Adding empty translations to database

### ISSUE THREAD

**Issue by skoTner** [spatie/laravel-translation-loader#86]
If the translation does not exist in the database or the translation files, the translation-string is returned instead. This is all and well, but there should be a parameter in the config-file (boolean) if we want these to be added to the database.

Then, when creating a user interface for editing translations I can automatically see the ones not translated yet, but used in the code. I could even make a report the the users of my system which translations not translated yet.

`trans('general.welcome');`

Not found in the database, so a row is added with the translation 'general.welcome'.

**Comment by freekmurze** [comment]
I don't think we can accomplish this in this package. It would be nice if Laravel fired an event when `trans` does not find a translation. You could listen for that event to do some project specific work.

**Comment by robov** [comment]
I created an override that adds the missing key to the table. However it seems that this key is added every single time to the database . This might have to do with that the database is updated, but the new data is not loaded in the cache.
Is there a way that I can reload the cache after I changed the database ?

When I add the translations in the seed, then everything goes well,
When I add the translations form within the missing key code, then it is still not found when I reload the page.
Any suggestions ?
